layers/meta-tegra-support/dynamic-layers/meta-mender-tegra/recipes-mender/tegra-state-scripts/tegra-state-scripts_%.bbappend

```diff
@@ -1 +1,20 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_remove = "file://redundant-boot-rollback-script-uboot"
+
+copy_install_script() {
+    sed -e's,@COPY_MACHINE_ID@,${PERSIST_MACHINE_ID},' ${S}/redundant-boot-install-script > ${MENDER_STATE_SCRIPTS_DIR}/ArtifactInstall_Leave_80_bl-update
+}
+
+copy_install_script_mender-uboot() {
+    cp ${S}/redundant-boot-install-script-uboot ${MENDER_STATE_SCRIPTS_DIR}/ArtifactInstall_Leave_80_bl-update
+    cp ${S}/redundant-boot-commit-check-script-uboot ${MENDER_STATE_SCRIPTS_DIR}/ArtifactCommit_Enter_80_bl-check-update
+}
+
+copy_other_scripts() {
+    :
+}
+
+do_compile_tegra210() {
+    cp ${S}/redundant-boot-install-script-uboot ${MENDER_STATE_SCRIPTS_DIR}/ArtifactInstall_Leave_80_bl-update
+}
```

Comment:
Same as above

Reply:
Hi @ichergui 
I already resolved all these comment with the latest commit done an hour ago.
Thanks

Reply:
Sorry tirankas
I didn't refresh the PR. 
Thanks for the update